### DISCUSSION TITLE
[companion] make comments and article accessible while writing

### DISCUSSION THREAD

**Discussion by GrahamTheDev** [dailydotdev/daily#604] [Feature Requests]
When commenting, you cannot see previous comments without closing the comment modal. 

Maybe save a draft of comments so you can switch back and forth between comments and the comment modal.

**Comment by idoshamun** [comment]
Hi @GrahamTheDev,

This is a dup of https://github.com/dailydotdev/daily/discussions/615.
Let's keep the discussion there.
